fsutils/afero.go

```diff
@@ -0,0 +1,42 @@
+package fsutils
```

Comment:
consider calling this `vfsutils` as it seems to be oriented towards virtual filesystems